RadarSDK/Include/RadarTrackingOptions.h

```diff
@@ -154,6 +154,16 @@ typedef NS_ENUM(NSInteger, RadarTrackingOptionsSyncLocations) {
  */
 @property (nonatomic, assign) BOOL beacons;
 
+/**
+ Either `default` if these options were set by the server, `on-trip` if they were set specifically for using during trips, or `nil` if they weren't set by the server.
+ */
+@property (nonatomic, copy) NSString *optionsType;
```

Comment:
We'll want to make the naming consistent with Android. I'm not sure "type" is the clearest thing